app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
     pass
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: float,
+                 average_rating: float, count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = round(count_of_ratings, 1)
```

Comment:
The `count_of_ratings` should not be rounded. It represents a count and should be an integer. Using `round` here is unnecessary and could lead to incorrect behavior.